templates/connect.php

```diff
@@ -325,9 +357,18 @@ class="fs-permission fs-<?php esc_attr_e( $id ); ?>">
 		 * @author Vova Feldman (@svovaf)
 		 * @since 1.1.9
 		 */
-		var $connectLicenseModeTrigger = $('#fs_connect .fs-freemium-licensing a');
+		var
+			$connectLicenseModeTrigger = $('#fs_connect .fs-freemium-licensing a'),
+			href = window.location.href;
+
+		if ( -1 === href.indexOf( '?' ) ) {

```

Comment:
@vovafeldman Code updated. I used `if ( href.indexOf( '?' ) > 0 )` instead of `if ( 0 < href.indexOf( '?' ) )` since the latter is hard to read and WP's Coding Standards page also mentions:
>Yoda conditions for <, >, <= or >= are significantly more difficult to read and are best avoided.